### ISSUE TITLE
Coding Guideline: Convert all tabs to spaces

### ISSUE THREAD

**Issue by theelderwand** [lukebuehler/CShell#80]
This is just a starter item. It should be part of a bigger engineering improvement around coding guidelines using StyleCop. Please let me know if this makes sense, i can take this up.

**Comment by lukebuehler** [comment]
Yes adding StyleCop would be great! If you want to take it, would be awesome.

**Comment by theelderwand** [comment]
This is turning out to be pretty big. Let me start off with one project get your opinions on them and then proceed ahead.
